src/main/java/tests/uiTests/CheckFormatForSelectedStatesTest.java

```diff
@@ -0,0 +1,21 @@
+package tests.uiTests;
+
+import domain.pages.DirectoryPage;
+import domain.pages.HomePage;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.support.PageFactory;
+
+public class CheckFormatForSelectedStatesTest extends UITestBaseClass{
+
+    @Test
+    public void checkFormat(){
```

Comment:
testnames include should
add test case description as java doc

Reply:
java doc done. All method names changed into proper format. But did not use should, need more clarification .on that.